### DISCUSSION TITLE
Save configuration

### DISCUSSION THREAD

**Discussion by xidus90** [AIrjen/OneButtonPrompt#76] [Ideas]
Can you add a function to save configurations (presets) or load parameters from the last session?

**Comment by AIrjen** [comment]
Hi! You can set a default yourself in the ui-config.json file of WebUI. This files contains all the default values used, and every extension is automatically part of this.
So you can set at least a default for yourself.

![image](https://github.com/AIrjen/OneButtonPrompt/assets/130234949/dd7d16bb-3be0-4503-85b5-c4236b438dce)

It is a good idea, and I will put the request on the todo list, but it does look like a lot of work. I'm not giving this the highest priority at the moment, got some other things I want to accomplish first.
